cli/src/psp-utils/createRustVerifyingKey.js

```diff
@@ -0,0 +1,391 @@
+"use strict";
```

Comment:
Why is this a new file? Did we have to move it from somewhere else?

Reply:
I believe tsc compilation creates these files in the same directory as the file if no output directory is specified.
I imported it by relative path in circuit-lib/circuit-lib.circom to avoid yarn link issues.
See:
https://github.com/Lightprotocol/light-protocol/blob/main/circuit-lib/circuit-lib.circom/scripts/createRustVerifyingKey.ts

@sergeytimoshin we could try to add the cli as a dev dependency to circuit-lib.circom maybe it will avoid this.
Or check whether an output dir is specified in circuit-lib.circom.

Reply:
Another option we could try is to move createVerifyingkeyRsFileArgv to zk.js or prover.js